### DISCUSSION TITLE
Debtor name character limit

### DISCUSSION THREAD

**Discussion by licsth** [schoero/SwissQRBill#390] [General]
Hi everyone,

I'm currently using this package to generate qr invoices that go to a lot of business clients, so the debtor name can become pretty lengthy (e.g. something like "Great Big Company GmbH, Manufacturing and Human Resources department c/o Alex Boss Chef") to the point where we regularly hit the character limit of 70 characters. I was wondering if there is a specific reason the limit is set to 70, or if it might be possible to extend it to something around 80-90?

Thanks for your help!
Linda

**Comment by schoero** [comment]
Hi Linda

Unfortunately this can not be changed. The Limitation comes from the [official QR-Bill specifications (page 30)](https://www.paymentstandards.ch/dam/downloads/ig-qr-bill-en.pdf).

Extending it would result in rendering issues and invalid QR-Codes that can not be scanned by banks.

Roger

**Reply by licsth** [reply]
Ok, thanks for the reply!
